src/app/components/navbar/components/message/message.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="wrapper bad" *ngIf="showing">
+<div class="wrapper bad" *ngIf="showBanner === true">
```

Comment:
Just curious, why doesn't `*ngIf="showBanner"` work?

Reply:
For some reason it was always evaluating to true even if the variable was set to false.